style.css

```diff
@@ -1,387 +1,517 @@
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
 }
 
-form {
+.contact {
+  position: relative;
+  padding: 6.4375rem 1.5rem 1.5rem;
+  z-index: 10;
   display: flex;
   flex-direction: column;
-  padding: 24px;
+  align-items: center;
 }
 
-input {
-  height: 48px;
-  width: 327px;
-  border: 1px solid #60c095;
-  box-sizing: border-box;
-  border-radius: 4px;
-  margin-bottom: 20px;
+.contact__title {
+  color: #fff;
+  text-align: center;
 }
 
-input::placeholder {
-  padding: 16px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
+.contact__dsc {
+  color: #ebebff;
+  text-align: center;
+  margin: 0.8rem 0 2.8rem;
 }
 
-textarea {
-  border: 1px solid #60c095;
-  box-sizing: border-box;
-  border-radius: 4px;
-  width: 327px;
-  margin-top: 18px;
+.contact__container form li input {
+  width: 100%;
+  border-radius: 8px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  margin: 10px 0;
+  height: 48px;
+  padding: 0 1rem;
 }
 
-textarea::placeholder {
-  padding: 15px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  background: #fff;
+.contact__container form li textarea {
+  width: 100%;
   border-radius: 8px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  margin-top: 8px;
+  padding: 1rem;
+  resize: none;
 }
 
-.submitbutton {
-  margin-top: 22px;
-  position: static;
-  width: 140px;
+.contact__container form li .btn {
+  width: 162px;
   height: 48px;
-  display: flex;
-  flex-direction: row;
-  padding: 12px 16px;
-  background: #fff;
-  border-radius: 8px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
+  padding: 12px;
   color: #6070ff;
+  cursor: pointer;
+  margin: 18px 0;
+}
+
+@media (min-width: 992px) {
+  .navbar {
+    padding: 1.5rem 8rem;
+  }
+
+  .navbar__menu {
+    display: flex;
+    align-items: center;
+  }
+
+  .navbar__menu__li {
+    margin: 0 16px;
+  }
+
+  .navbar__hamb {
+    display: none;
+  }
+
+  .head::before {
+    background-image: url("./logos/headershape.png");
+    padding: 6rem;
+    border-bottom-left-radius: 8rem;
+  }
+
+  .head {
+    padding: 10rem;
+    padding-right: 15rem;
+  }
+
+  .head__title span {
+    width: 15ch;
+  }
+
+  .head__link {
+    margin: 0.7rem 0;
+  }
+
+  .works__cards {
+    padding: 5rem;
+  }
+
+  .works__card {
+    max-width: 1104px;
+    -moz-column-gap: 46px;
+    column-gap: 46px;
+    padding: 1.5rem;
+  }
+
+  .works__card__text {
+    padding: 1rem;
+  }
+
+  .works__card > * {
+    flex-basis: 50%;
+  }
+
+  .works__card:nth-child(1),
+  .works__card:nth-child(3) {
+    flex-direction: row;
+  }
+
+  .works__card:nth-child(2),
+  .works__card:nth-child(4) {
+    flex-direction: row-reverse;
+  }
```

Comment:
- [ Optional ] It is nice that you have used this selector to style the card dynamically. But, to make it scale as you add more projects later on, wouldn't it be great if you could use the `nth-child(odd)` and `nth-child(even)` selectors? Please take a look at this [example](https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_nth-child_odd_even).